jsonenums_test.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"testing"
+)
+
+func Test_ToSnake(t *testing.T) {
+	sample1 := "aBcDEfg"
+	if "a_bc_d_efg" != ToSnake(sample1) {
```

Comment:
Better than nothing, but the table of tests you initially linked seemed like it was very comprehensive. Was there a reason to not replicate that here?